alpha/apps/kaltura/lib/batch2/bcdl/kBusinessPostConvertDL.php

```diff
@@ -321,7 +321,14 @@ public static function handleConvertFinished(BatchJob $dbBatchJob = null, flavor
     			{
     				if($srcFlavor->getType() == assetType::FLAVOR)
     				{
+    					KalturaLog::info('Export source asset to private and periodic storages');
+
+    					//Private storage
     					kStorageExporter::handleAssetStorageExports($srcFlavor);
+
+    					//Periodic storage
+    					$periodicStorageProfiles = kStorageExporter::getPeriodicStorageProfiles($srcFlavor->getPartnerId());
```

Comment:
No need to add more mess to this huge function, just call one function that handles both cases (private and periodic) inside kStorageExporter.